local_settings.example.py

```diff
@@ -257,6 +265,36 @@
 
 # }}}
 
+# {{{ overriding built-in templates and staticfiles
+# Uncomment the following to enable templates overriding. It should be configured
+# as a list/tuple of path(s).
+# For example, if you the templates are in a folder named "my_templates" in the
+# root dir of the project, with base.html (project template), course_base.html,
+# and sign-in-email.txt (app templates) etc., are the templates you want to
+# override, the structure of the files should look like:
+#    ...
+#    relate/
+#    local_settings.py
+#    my_templates/
+#        base.html
+#        ...
+#        templates/
+#            course/
+#                course_base.html
+#                sign-in-email.txt
+#                ...
+#
+
+# import os.path
+# MY_TEMPLATES_DIRS = [os.path.join(os.path.dirname(__file__), "my_templates"),
+#                      os.path.join(os.path.dirname(__file__), "my_other_templates")]
+
+# Uncomment the following to enable staticfile overriding, It should be configured
+# as a list/tuple of path(s)
+
+# MY_STATICFILES_DIRS = [os.path.join(os.path.dirname(__file__), "staticfiles"),]
```

Comment:
Doesn't Django already have a way of overriding these? If so, is it really necessary to add these?

Reply:
Yes, the overriding of staticfiles works like that of Django templates. These are shortcut for overriding, it make it easier especially for template (there will be less work when settings.py is updated), and I agree it's less necessary for staticfiles. I added that mainly because #355 is adding staticfiles to the project for the new theme. I'll remove it.

Reply:
For staticfiles, perhaps it's only necessary when more work can be done on proposals like #355.

Reply:
Removed.